app/models/camp.rb

```diff
@@ -119,6 +122,11 @@ class Camp < ActiveRecord::Base
     where(is_public: flag)
   }
 
+  scope :is_cocreation, lambda { |flag|
```

Comment:
I'm not sure what this scope does. Why do you accept an argument if you don't use it?
If you're looking for non-null values, not-equal to a certain value, you can simplify this to:
`where.not(camps: { cocreation: nil }).where.not(camps: { cocreation: '" ' })`
What does `" ` mean?